--- Framework.py.orig	2025-09-09 11:56:53.659445709 +0200
+++ Framework.py	2025-09-09 11:58:27.196414016 +0200
@@ -345,12 +345,13 @@
 
 class BasicTestCase(unittest.TestCase):
     recordMode = False
+    recordActionMode = False
     replayDataFolder = os.path.join(os.path.dirname(__file__), "ReplayData")
 
     def __init__(self, methodName="runTest") -> None:
         super().__init__(methodName)
         self.authMode = "token"
-        self.per_page = Consts.DEFAULT_PER_PAGE
+        self.per_page = 100
         self.retry = None
         self.pool_size = None
         self.seconds_between_requests: float | None = None
@@ -520,7 +521,6 @@
             auth=auth,
             per_page=self.per_page,
             retry=retry,
-            pool_size=pool_size,
             seconds_between_requests=self.seconds_between_requests,
             seconds_between_writes=self.seconds_between_writes,
         )
@@ -528,3 +528,7 @@
 
 def activateRecordMode():  # pragma no cover (Function useful only when recording new tests, not used during automated tests)
     BasicTestCase.recordMode = True
+
+
+def activateRecordActionMode():  # pragma no cover (Function useful only when recording new tests, not used during automated tests)
+    BasicTestCase.recordActionMode = True
