--- Framework.py.orig	2025-08-28 09:50:04.002076656 +0200
+++ Framework.py	2025-08-28 09:50:39.017704705 +0200
@@ -344,8 +344,9 @@
 
 class BasicTestCase(unittest.TestCase):
     recordMode = False
+    recordActionMode = False
     authMode = "token"
-    per_page = Consts.DEFAULT_PER_PAGE
+    per_page = 100
     retry = None
     pool_size = None
     seconds_between_requests: float | None = None
@@ -497,7 +498,6 @@
             auth=auth,
             per_page=self.per_page,
             retry=retry,
-            pool_size=pool_size,
             seconds_between_requests=self.seconds_between_requests,
             seconds_between_writes=self.seconds_between_writes,
         )
@@ -517,3 +517,7 @@
 
 def setPoolSize(pool_size):
     BasicTestCase.pool_size = pool_size
+
+
+def activateRecordActionMode():  # pragma no cover (Function useful only when recording new tests, not used during automated tests)
+    BasicTestCase.recordActionMode = True
